package/suse-migration-services-spec-template

```diff
@@ -26,6 +26,7 @@ Group:            System/Management
 Source:           suse-migration-services.tar.gz
 BuildRoot:        %{_tmppath}/%{name}-%{version}-build
 Conflicts:        suse-migration-sle15-activation
+Conflicts:        suse-migration-pre-checks
```

Comment:
The pre-checks and the migration services both need to be installed in the Live ISO system that does the migration, they cannot conflict.